gateway/reverse_proxy.go

```diff
@@ -1136,9 +1138,10 @@ func (p *ReverseProxy) handoverRequestToGraphQLExecutionEngine(roundTripper *Tyk
 		}
 
 		isProxyOnly := isGraphQLProxyOnly(p.TykAPISpec)
-		reqCtx := context.Background()
+		span := tyktrace.SpanFromContext(outreq.Context())
+		reqCtx := oteltrace.ContextWithSpan(context.Background(), span)
```

Comment:
@kofoworola Can we create this function on https://github.com/TykTechnologies/opentelemetry/blob/main/trace/span.go and import it in the `internal/otel` pkg?

Reply:
https://github.com/TykTechnologies/opentelemetry/pull/22
PR for this